src/DblParser/Desugar.ml

```diff
@@ -660,17 +666,17 @@ and tr_def ?(public=false) (def : Raw.def) =
   match def.data with
   | DLet(pub, p, e) ->
     let public = public || pub in
-    begin match tr_let_pattern ~public p with
+    [ match tr_let_pattern ~public p with
     | LP_Id id -> 
-      make (DLetId(id, tr_expr e))
+        make (DLetId(id, tr_expr e))
     | LP_Fun(id, targs, iargs, args) ->
-      make (DLetFun(id, targs, iargs, tr_function args (tr_expr e)))
+        make (DLetFun(id, targs, iargs, tr_function args (tr_expr e)))
     | LP_Pat p ->
-      make (DLetPat(p, tr_expr e))
-    end
+        make (DLetPat(p, tr_expr e))
+    ]
```

Comment:
I'm not sure if I like this style of code formatting. Please consider leaving `begin match .. end` and putting braces around `make (...)`. Or at least indent match clauses to align them with `match` keyword. Same comment applies to some other changes.